src/commons/sideContent/content/SideContentCseMachine.tsx

```diff
@@ -200,24 +194,23 @@ class SideContentCseMachineBase extends React.Component<CseMachineProps, State>
   }
 
   public render() {
-    const cseMachineHandlers = this.state.visualization
-      ? {
-          FIRST_STEP: this.stepFirst,
-          NEXT_STEP: this.stepNext,
-          PREVIOUS_STEP: this.stepPrevious,
-          LAST_STEP: this.stepLast(this.props.stepsTotal)
-        }
-      : {
-          FIRST_STEP: () => {},
-          NEXT_STEP: () => {},
-          PREVIOUS_STEP: () => {},
-          LAST_STEP: () => {}
-        };
+    const hotkeyBindings: HotkeyItem[] = this.state.visualization
+      ? [
+          ['a', this.stepFirst],
+          ['f', this.stepNext],
+          ['b', this.stepPrevious],
+          ['e', this.stepLast(this.props.stepsTotal)]
+        ]
+      : [
+          ['a', () => {}],
+          ['f', () => {}],
+          ['b', () => {}],
+          ['e', () => {}]
+        ];
```

Comment:
While not related to this PR, it it just be or are the keyboard shortcuts not intuitive? I would expect the following:

(first, prev, next, last) to be mapped to (h, j, k, l) or (a, s, d, f) -- basically just a sequence of keys that are next to each other on the keyboard.